prow/cmd/deck/static/template/spyglass-template.html

```diff
@@ -126,6 +126,15 @@
         document.getElementById(skipID).style.display = "none";
       }
 
+      function toggleExpansion(bodyId, expanderId) {
```

Comment:
I don't love the idea of leaking viewer-specific functionality into the main spyglass template.

Reply:
Absolutely agree, thanks for linking the issue.

Reply:
That issue is a big one, I've started working on a fix for it but still ironing out the kinks